### PR TITLE
TrayPublisher: implemented render_mov_batch 

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Re-implementation of `render_mov_batch` family for TrayPublisher.

## Description
This family is used to publish multiple .mov files at once. Artists doesn't need to select context, some heuristic is implemented to parse asset name from a file name (`chair.mov`, `chair_v001.mov`...).
Resulting task name is configured in Settings. (Could be configured multiple ['Compositing', 'Art'] - first available task on Asset will be picked up.

## Testing notes:
1. Upload single or multiple .mov files in TrayPublisher.
2. Settings default contains 'Compositing' as a resulting task name
3. Try to publish .mov file with dummy name, it should warn you, that file name doesn't contain valid asset name

### CODE REVIEW

File: openpype/hosts/traypublisher/plugins/create/create_mov_batch.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+from openpype.hosts.traypublisher.api.plugin import TrayPublishCreator
+
+
+class BatchMovCreator(TrayPublishCreator):
+    """Creates instances from .mov file(s)."""
+    identifier = "render_mov_batch"
+    label = "Batch Mov"
+    family = "render"
+    description = "Publish batch of movs"
+    host_name = "traypublisher"
```
Comment (iLLiCiTiT) [inline openpype/hosts/traypublisher/plugins/create/create_mov_batch.py:21]:
```suggestion
    host_name = "traypublisher"
```
Already defined from `TrayPublishCreator`

File: openpype/hosts/traypublisher/plugins/create/create_mov_batch.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                                              *args, **kwargs)
+        self.default_variants = (project_settings["traypublisher"]
+                                                 ["BatchMovCreator"]
+                                                 ["default_variants"])
+
+        self.default_tasks = (project_settings["traypublisher"]
+                                              ["BatchMovCreator"]
+                                              ["default_tasks"])
+
+        self.extensions = (project_settings["traypublisher"]
```
Comment (iLLiCiTiT) [inline openpype/hosts/traypublisher/plugins/create/create_mov_batch.py:37]: I'm not sure if we should allow to modify extensions for this creator.

Comment (mkolar) [reply openpype/hosts/traypublisher/plugins/create/create_mov_batch.py:37]: The other option would be to include pretty much all video and image formats we can think of out of the box. Not sure that's better. I'd leave it in the settings personally

File: openpype/hosts/traypublisher/plugins/create/create_mov_batch.py [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+                                           ["BatchMovCreator"]
+                                           ["extensions"])
```
Comment (iLLiCiTiT) [inline openpype/hosts/traypublisher/plugins/create/create_mov_batch.py:39]: Maybe prepare variable with this plugin settings
```suggestion
        creator_settings = (
            project_settings["traypublisher"]["BatchMovCreator"]
        )
        self.default_variants = creator_settings["default_variants"]
        self.default_tasks = creator_settings["default_tasks"]
        self.extensions = creator_settings["extensions"]
```

File: openpype/settings/entities/schemas/projects_schema/schema_project_traypublisher.json [hunk 4]
```diff
@@ -78,6 +78,80 @@
...
+        },
+         {
+            "type": "dict",
+            "collapsible": true,
+            "key": "BatchMovCreator",
+            "label": "Batch Mov Creator",
+            "use_label_wrap": true,
+            "collapsible_key": true,
+            "children": [
+                {
```
Comment (iLLiCiTiT) [inline openpype/settings/entities/schemas/projects_schema/schema_project_traypublisher.json:90]: There are unused settings `family`. `identifier`, `label`, `icon`, `description` and `detailed_description`.

File: openpype/hosts/traypublisher/plugins/publish/collect_mov_batch.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    on creator.
+    """
+
+    label = "Collect Mov Batch Files"
+    order = pyblish.api.CollectorOrder
+
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        if not instance.data.get("creator_identifier") == "render_mov_batch":
```
Comment (iLLiCiTiT) [inline openpype/hosts/traypublisher/plugins/publish/collect_mov_batch.py:22]:
```suggestion
        if instance.data.get("creator_identifier") != "render_mov_batch":
```

**Comment by iLLiCiTiT** [comment]
There is unhandled case when subset name requires task. In that case instance won't be created to be able set it.

**Review by iLLiCiTiT — APPROVED** [review]
Looks good and works.

**Pull request merged**